modules/operators-overview.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='operators-overview-{context}']
+= Operators in {product-title}
+
+In {product-title} version 4.0, all cluster functions are divided into a series
+of component Operators.
```

Comment:
Platform Operators.  Platform operators manage a particular area of functionality like cluster wide application logging, the management of the Kubernetes control plane, or the machine provisioning system.